lizzy/apps/senza.py

```diff
@@ -1,17 +1,20 @@
 from typing import Optional, List, Dict
 import tempfile
 
+from ..version import VERSION
```

Comment:
@jmcs I decided to use a unique lizzy version. I am not sure why we had two different version formats. I sent a PR (https://github.com/zalando/lizzy-client/pull/32) to `lizzy-client` to use new format too.